lib/features/add_property/view/add_features_view.dart

```diff
@@ -0,0 +1,213 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/dimens/hs_dimens.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/strings/l10n.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+
+const EdgeInsets _kScreenPadding =
+    EdgeInsets.symmetric(horizontal: HsDimens.lineheight16);
+
+class AddFeaturesView extends StatefulWidget {
+  const AddFeaturesView({super.key});
+
+  @override
+  State<AddFeaturesView> createState() => _AddFeaturesViewState();
+}
+
+class _AddFeaturesViewState extends State<AddFeaturesView> {
+  final ValueNotifier<List<String>> selectedFeatures =
+      ValueNotifier<List<String>>([]);
+
+  @override
+  void dispose() {
+    super.dispose();
+    selectedFeatures.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        const SizedBox(
+          height: HsDimens.lineheight24,
+        ),
+        Padding(
+          padding: _kScreenPadding,
+          child: Text(
+            HatSpaceStrings.of(context).askFeaturesOwned,
+
+            /// This one help preventing overflow when run flutter test
+            maxLines: 3, overflow: TextOverflow.ellipsis,
+            style: Theme.of(context)
+                .textTheme
+                .displayLarge
+                ?.copyWith(color: HSColor.neutral9),
+          ),
+        ),
+        const SizedBox(
+          height: HsDimens.spacing20,
+        ),
+        Expanded(
+          child: ValueListenableBuilder(
+            valueListenable: selectedFeatures,
+            builder: (_, selectedFeatures, __) {
+              return GridView.count(
```

Comment:
Create an enum of all features we have, and use GridView.builder with values of that enum to build the grid

Reply:
@suesitran Can you be more details ? You mean like this ? And inside builder of GridView will use switch or if else right ?

```
enum Feature {
    fridge, 
    swimming pool, 
    .....
} 
```

Reply:
```
enum Feature {
    fridge(HatSpaceStrings.current.fridge, Assets.images.fridge), 
    swimming pool(HatSpaceStrings.current.swimmingpool, Assets.image.swimmingPool), 
    .....,

   final String displayName;
   final String iconSvgPath;
} 
```